uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/SamsungSettings.java

```diff
@@ -0,0 +1,186 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation.device.settings;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.test.uiautomator.UiObject;
+import androidx.test.uiautomator.UiObjectNotFoundException;
+
+import com.microsoft.identity.client.ui.automation.broker.ITestBroker;
+import com.microsoft.identity.client.ui.automation.utils.AdbShellUtils;
+import com.microsoft.identity.client.ui.automation.utils.UiAutomatorUtils;
+
+import org.junit.Assert;
+
+import java.util.Calendar;
+
+public class SamsungSettings extends BaseSettings {
+
+    @Override
+    public void disableAdmin(@NonNull final String adminName) {
+        launchDeviceAdminSettingsPage();
+
+        try {
+            // scroll down the recycler view to find the list item for this admin
+            final UiObject adminAppListItem = UiAutomatorUtils.obtainChildInScrollable(
+                    "android:id/list",
+                    adminName
+            );
+
+            // Click into this admin
+            assert adminAppListItem != null;
+            adminAppListItem.click();
+
+            // Click Deactivate button
+            UiAutomatorUtils.handleButtonClick("com.android.settings:id/action_button");
+
+            if ("Company Portal".equalsIgnoreCase(adminName)) {
```

Comment:
Opportunity to have an `AdminNames` object or constant?

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/982/commits/087aa79b92bf86d15b01ac0623aa3c4b316507e7